packages/modules/src/modules/banners/supporterMoment/SupporterMomentBanner.stories.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import { brand, culture } from '@guardian/src-foundations';
+import { bannerWrapper } from '../common/BannerWrapper';
+import { getMomentTemplateBanner } from '../momentTemplate/MomentTemplateBanner';
+import { BannerProps } from '@sdc/shared/src/types';
+import { Meta, Story } from '@storybook/react';
+import { props } from '../utils/storybook';
+
+export default {
+    title: 'Banners/MomentTemplate',
+    parameters: {
+        chromatic: {
+            delay: 300,
+        },
+    },
```

Comment:
I'm not sure this parameter is necessary anymore. I believe it was added as a temporary measure during the render delay test

Reply:
All storybook banners have this delay, should I remove all, perhaps in a separate cleanup PR might be better? There is one called 'RBPD-amounts-definition-move' which contains some cleanups that may be better to put into?

Reply:
Can do that now.

Reply:
Ah okay if they were all updated at once then would be best to do this separately